src/index.js

```diff
@@ -59,14 +58,59 @@ function cleanupAtContainer(container) {
   mountedContainers.delete(container)
 }
 
+const originalChange = fireEvent.change
+fireEvent.change = function reactChange(node, init) {
+  if (init && init.target && init.target.hasOwnProperty('value')) {
+    setNativeValue(node, init.target.value)
+  }
+  return originalChange(node, init)
+}
+
+// probably there is a better way to get access to a component instance
+const [
+  getInstanceFromNode,
+] = ReactDOM.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED.Events
+const createReactEventSimulator = eventName => (node, init) => {
```

Comment:
Calling the prop directly sorta works, but I _think_ I'd rather come up with a way to fire the browser events that react is looking for when it triggers the callback.

Reply:
OK, I'll dig into that deeper 😄